magstock/templates/registration/check_in_form.html

```diff
@@ -0,0 +1,16 @@
+{% extends "registration/check_in_form_base.html" %}
+{% block checkin_fields %}
+    <tr>
+        <td><strong>Food?</strong></td>
+        <td>{{ attendee.gets_food }}</td>
```

Comment:
only other totally optional minor thing is if this can say "give food wristband" (which is how we denote someone gets food), just to make it really obvious that the reg staffer should give someone a physical thing in response to this.